KITE-Extras/src/main/java/io/cosmosoftware/kite/util/WebDriverInteractions.java

```diff
@@ -0,0 +1,399 @@
+/*
```

Comment:
this will require our sample tests refactoring

Reply:
Not necessarily, these functions has webdriver as parameter, and are public static. The pages can still use functions in BasePage